files-community/Files
# Fix: Fixed cannot select Files in columns view

**Resolved / Related Issues**
   Closes #12090 

**Description**
- I simplified the code to find the last column that should remain open (I don't know why I wrote that complex code in the first place). Now it looks for the blade with the same path as the requested one.
- As mentioned in #11898, there's a bug with double click to open

**Notes**
- Please, test these changes deeply since they may cause many regressions
- It may happen that the user creates a loop of columns (like: `C:\Users\username\Desktop\shortcut_to_Users\username\Desktop\something_else`). In this case, if the user clicks an item in `Desktop`, should we always close all columns after the first `Desktop`?

**Validation**
How did you test these changes?
**Test 1**
   1. Open app and a folder using colums view
   2. Use `Open folders with single click in columns view`
   3. Open a shortcut to a folder (try with a shortcut to a sub-subfolder of the current directory, a shortcut to a parent folder of the current and with one to a random folder)
   4. Select a file
**Test 2**
   1. Repeat all the steps above using `double click to open`
   2. Navigate between folders using `Up/Down arrows` to ensure there's no regression from #11986   
**Test 3**
   1. Open a path such as C:\Windows\System32 in columns layout
   2. Click on Windows (Single click to open option) (Check that there's no regression from #11898)
